barricade/src/main/java/com/mutualmobile/barricade/activity/BarricadeActivity.java

```diff
@@ -116,4 +121,19 @@ private void showEditDialog() {
     EditGlobalDelayDialogFragment dialogFragment = new EditGlobalDelayDialogFragment();
     dialogFragment.show(fm, EditGlobalDelayDialogFragment.class.getName());
   }
+
+  private void showResetDialog() {
+    AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
Continue the builder. Also .show() internally calls .create() so no need of that